k8s/prod/common/ccd/data-store-api.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccd-data-store-api
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccd-data-store-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccd-data-store-api
+    version: 1.2.4
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ccd/data-store-api:prod-a1b64606
```

Comment:
you have to override idam_url in prod config . also please check prod.tfvars and values.yaml if any other variables need to be override